mne/connectivity/envelope.py

```diff
@@ -107,22 +126,28 @@ def envelope_correlation(data, combine='mean', orthogonalize="pairwise",
         corr = np.empty((n_nodes, n_nodes))
         for li, label_data in enumerate(epoch_data):
             if orthogonalize is False:  # the new code
-                label_data_orth = data_mag
-                label_data_orth_std = data_mag_std
+                label_data_orth = data_mag[li]
+                label_data_orth_std = data_mag_std[li]
             else:
                 label_data_orth = (label_data * data_conj_scaled).imag
+                np.abs(label_data_orth, out=label_data_orth)
+                if log:
+                    label_data_orth *= label_data_orth
+                    np.log10(label_data_orth, out=label_data_orth)
                 label_data_orth -= np.mean(label_data_orth, axis=-1,
                                            keepdims=True)
                 label_data_orth_std = np.linalg.norm(label_data_orth, axis=-1)
                 label_data_orth_std[label_data_orth_std == 0] = 1
             # correlation is dot product divided by variances
-            corr[li] = np.dot(label_data_orth, data_mag_nomean[li])
-            corr[li] /= data_mag_std[li]
+            corr[li] = np.sum(label_data_orth * data_mag_nomean, axis=1)
```

Comment:
Second change is what @Avoide talked about in the other issue, we need to correlate with the correct pairs. This (plus the `log=True, absolute=False`) makes our code match both what @Avoide suggested and what FieldTrip produces.

Reply:
We were using cor, but this is much faster

> Second change is what @Avoide talked about in the other issue, we need to correlate with the correct pairs. This (plus the `log=True, absolute=False`) makes our code match both what @Avoide suggested and what FieldTrip produces.

We were using np.corrcoef but this is much faster